tests/should-fail/iati-activities/iati-activity/result/indicator/_aggregation-status/err-not-iati-xml-incorrect-datatype.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity> <!-- at least 1 -->
+    <iati-identifier></iati-identifier>  <!-- 1 and only 1-->
+    <reporting-org type="xx" ref="xx"><narrative>Organisation name</narrative></reporting-org> <!-- 1 and only 1-->
+    <title> <!-- 1 and only 1-->
+      <narrative>Xxxxxxx</narrative> <!-- At least 1-->
+    </title>
+    <description>
+      <narrative>Xxxxxxx</narrative> <!-- At least 1-->
+    </description>
+    <participating-org role="xx"></participating-org> <!-- At least 1-->
+    <!--<other-identifier></other-identifier>-->
```

Comment:
All of the comments unrelated to the thing being tested makes it hard to see what the important part of the dataset is.

Reply:
Or we could prepend comments with `INFO:` and `ERROR:`, for example:

    <activity-date type="xx" iso-date="2013-11-27"><!-- INFO: At least 1 --> <!-- INFO: Narative (sic) allowed-->
       <narrative>Some stuff ere</narrative>
     </activity-date>
     <result type="xx">
       <title>
         <narrative>Sample text</narrative>
       </title>
       <indicator measure="xx" aggregation-status="xx"><!-- ERROR: optional aggregation-status attribute included with non-permitted value -->
```

Reply:
Definitely looks better, though it should also be made sure that the comments are relevant.

For example, `INFO: Narative (sic) allowed` in that example is completely unrelated and irrelevant to what the test file is checking - that a non-boolean value in a boolean attribute fails validation.

(as a side-note, that name is awful for a boolean value since it gives little indication as to what each `true` and `false` mean - `aggregatable` is a word! >.<)

Reply:
This thread is now resolved by internal agreement to remove non-relevant comments and include prepended comments on all included comments to highlight relevant status.